common/strings/patch.cc

```diff
@@ -363,7 +367,8 @@ LineNumberSet FilePatch::AddedLines() const {
 static char PromptHunkAction(std::istream& ins, std::ostream& outs) {
   // Suppress prompt in noninteractive mode.
   if (IsInteractiveTerminalSession()) {
-    outs << "Apply this hunk? [y,n,a,d,s,q,?] ";
+    outs << term::StartColor(term::Color::kYellow)
```

Comment:
I think I'd avoid yellow, as it is really hard to see on terminals with white background. How about we use `term::bold()` here ?

Reply:
I agree, didn't think about this. Bold seems right for every type of background.